libraries/ms-drivers/src/x86/pca9539r_gpio_expander.c

```diff
@@ -81,3 +81,14 @@ StatusCode pca9539r_gpio_get_state(const Pca9539rGpioAddress *address,
   *input_state = s_pin_settings[address->i2c_address][address->pin].state;
   return STATUS_CODE_OK;
 }
+
+StatusCode pca9539r_gpio_subscribe_interrupts(const GpioAddress *interrupt_pin,
+                                              const Pca9539rInterruptCallback callback,
```

Comment:
The callback is declared as `const` here but not in the header file - it doesn't really matter whether it's const or not, but please be consistent.

Reply:
My IDE is telling me when I qualify the callback as const: 
Clang-Tidy: Parameter 'callback' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions

Reply:
I looked into this and there is a thread that talks more about this [here](https://stackoverflow.com/questions/51353648/const-qualification-of-parameters-in-function-declaration) if you're interested. You shouldn't need the const qualifier on the callback param though you can just get rid of it